face/capture/src/main/java/com/simprints/face/capture/models/ScreenOrientation.kt

```diff
@@ -0,0 +1,16 @@
+package com.simprints.face.capture.models
+
+import android.content.res.Configuration
+import android.content.res.Resources
+
+enum class ScreenOrientation {
```

Comment:
As per the latest guidelines, we should use https://developer.android.com/reference/androidx/window/core/layout/WindowSizeClass instead of the orientation.

Reply:
We only require the orientation from the resources (the layout being used by the system - regular one or the one from the `layouts/land` folder). Getting the orientation from the `resources` is exactly what is required in this case